<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapRulerProperties</class>
 <widget class="QWidget" name="MapRulerProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>28</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ruler Properties</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QToolButton" name="greatCircleMode">
     <property name="toolTip">
      <string>Great circle mode</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">rulerModeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="straightLineMode">
     <property name="toolTip">
      <string>Straight line mode</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">rulerModeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="headingButton">
     <property name="toolTip">
      <string>Display heading</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">labelModeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="rangeButton">
     <property name="toolTip">
      <string>Display distance</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">labelModeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="speedButton">
     <property name="toolTip">
      <string>Display relative speed</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">labelModeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="etaButton">
     <property name="toolTip">
      <string>Display ETA</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">labelModeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="labelSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Label size</string>
     </property>
     <property name="minimum">
      <number>6</number>
     </property>
     <property name="maximum">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="labelDestButton">
     <property name="toolTip">
      <string>Show label at destination</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">labelModeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="labelSrcButton">
     <property name="toolTip">
      <string>Show label at source</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">labelModeButtonGroup</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="labelModeButtonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="rulerModeButtonGroup"/>
 </buttongroups>
</ui>
